engine/table/src/main/java/io/deephaven/engine/table/impl/partitioned/LongConstantColumn.java

```diff
@@ -70,14 +70,9 @@ public SelectColumn copy() {
         return new LongConstantColumn(outputColumnName, outputValue);
     }
 
-    @Override
-    public final List<String> initInputs(@NotNull final Table table) {
-        return initInputs(table.getRowSet(), table.getColumnSourceMap());
-    }
-
     @Override
     public final Class<?> getReturnedType() {
-        return Table.class;
+        return long.class;
```

Comment:
Why should this class exist in addition to io.deephaven.engine.table.impl.sources.LongSingleValueSource#LongSingleValueSource?

Reply:
Two reasons I can come up with:
1. It is simpler and tighter, since it's immutable and has no previous tracking.
2. It can be used in a select/view/update/updateView with other `SelectColumn` instances. This is more powerful, and also way more accessible than having to build a custom copy operation that adds columns.

Reply:
We agreed to a small change here.